core/src/main/scala/com/salesforce/op/OpWorkflowModel.scala

```diff
@@ -466,6 +466,7 @@ case object OpWorkflowModel {
    * @param path to the trained workflow model
    * @return workflow model
    */
-  def load(path: String): OpWorkflowModel = new OpWorkflowModelReader(None).load(path)
+  def load(path: String, asSpark: Boolean = true): OpWorkflowModel =
```

Comment:
missing docs
`asSpark if true will load as spark models if false will load as Mlleap stages for spark wrapped stages`

Reply:
are we exposing `asSpark` to be overridable in production ?

Reply:
yes we will allow models to be loaded with and without spark